lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -122,6 +122,14 @@ export class PublicClientApplication {
         }
     }
 
+    /**
+     * Event handler function which allows users to fire events after the PublicClientApplication object 
+     * has loaded during redirect flows.
+     */
+    async onRedirectAppLoad(): Promise<TokenResponse> {
```

Comment:
Name proposal: `handleRedirectPromise` to keep it close to `handleRedirectCallback`

Reply:
done.